src/Symfony/Component/Routing/Route.php

```diff
@@ -266,23 +266,23 @@ public function setOption(string $name, $value)
     }
 
     /**
-     * @return mixed The option value or null when not given
```

Comment:
usefull to keep as method description

Reply:
why useful? `getOption($name)` returns the option $name, isn't that obvious?

Reply:
just because of "null when not given"

Reply:
OK,updated